SF iOS/SF iOS/AppDelegate.m

```diff
@@ -54,4 +56,16 @@ - (void)application:(UIApplication *)application performFetchWithCompletionHandl
         self.bgFetcher = nil;
     }];
 }
+
+//MARK: - Configure cache
+- (void)setSharedNetworkCacheMemoryMegabytes:(NSInteger)memoryMiB diskMegabytes:(NSInteger)diskMiB
+{
+    NSUInteger cashSize = memoryMiB * 1024 * 1024;
+    NSUInteger cashDiskSize = diskMiB * 1024 * 1024;
+    NSURLCache *imageCache = [[NSURLCache alloc] initWithMemoryCapacity:cashSize
+                                                           diskCapacity:cashDiskSize
+                                                               diskPath:@"networking"];
+    [NSURLCache setSharedURLCache:imageCache];
+}
```

Comment:
Not a huge deal, but architecturally I think there is a good argument that states that this belongs as a static method on an NSURLCache category.